docker/data-science/notebooks/ch10/1.ch10-setup-TheMovieDB.ipynb

```diff
@@ -0,0 +1,98 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append('..')\n",
+    "from aips import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create Collection\n",
+    "\n",
+    "Create collection for http://themoviedb.org (TMDB) dataset for this book. We will just look at title, overview, and release_year fields."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tmdb_collection=\"tmdb\"\n",
+    "create_collection(tmdb_collection)\n",
```

Comment:
Not working for me. I'm getting:

Wiping 'tmdb' collection
[('action', 'CREATE'), ('name', 'tmdb'), ('numShards', 1), ('replicationFactor', 1)]
Creating tmdb' collection
Status: Success
Del/Adding LTR QParser for tmdb' collection
Status: Failure; Response:[ {'responseHeader': {'status': 400, 'QTime': 8}, 'errorMessages': ['error processing commands\n'], 'WARNING': 'This response format is experimental.  It is likely to change in the future.', 'error': {'metadata': ['error-class', 'org.apache.solr.api.ApiBag$ExceptionWithErrObject', 'root-error-class', 'org.apache.solr.api.ApiBag$ExceptionWithErrObject'], 'details': [{'add-queryparser': {'name': 'ltr', 'class': 'org.apache.solr.ltr.search.LTRQParserPlugin'}, 'errorMessages': ["Error loading class 'org.apache.solr.ltr.search.LTRQParserPlugin'"]}], 'msg': 'error processing commands', 'code': 400}} ]
Adding LTR Doc Transformer for tmdb' collection
Status: Failure; Response:[ {'responseHeader': {'status': 400, 'QTime': 8}, 'errorMessages': ['error processing commands\n'], 'WARNING': 'This response format is experimental.  It is likely to change in the future.', 'error': {'metadata': ['error-class', 'org.apache.solr.api.ApiBag$ExceptionWithErrObject', 'root-error-class', 'org.apache.solr.api.ApiBag$ExceptionWithErrObject'], 'details': [{'add-transformer': {'name': 'features', 'class': 'org.apache.solr.ltr.response.transform.LTRFeatureLoggerTransformerFactory', 'fvCacheName': 'QUERY_DOC_FV'}, 'errorMessages': ["Error loading class 'org.apache.solr.ltr.response.transform.LTRFeatureLoggerTransformerFactory'"]}], 'msg': 'error processing commands', 'code': 400}} ]
Adding 'title' field to collection
Status: Success
Adding 'overview' field to collection
Status: Success
Adding 'release_year' field to collection
Status: Success